Caboodle/Browser/Browser.shared.enums.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Caboodle
+{
+    public enum BrowserLaunchingType
```

Comment:
Should this be in a separate file, or just next to the browser class?

Reply:
I have been making a .shared.enums.cs file to separate them. I kinda like it and they are grouped together.